websites/C/ChompuBot/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "919878532228841532",
+		"name": "somboytiger"
+	},
+	"service": "ChompuBot",
+	"description": {
+		"en": "Chompu is Discord bot music spport stream Joox, TikTok, Distrokid, Spotify, AppleMusic, Soundcloud, Clyp, PornHub and etc. Can req song on website/dashboard [TH]",
+		"th": "Chompu คือ บอทดิสคอร์ดที่มาพร้อมด้วยระบบสตรีมเล่นเพลงที่สามารถจัดการหรือเพิ่มคิวเพลงบนเว็บไซต์! รองรับสตรีมเล่นเพลงหลายแพลตฟอร์ม Joox, TikTok, Distrokid, Spotify, AppleMusic, Soundcloud, Clyp, PornHub และอื่นๆอีกมากมาย!"
+	},
+	"url": "chompubot.work",
+	"matches": [
+		"*://chompubot.work/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://chompubot.work/bot.png",
+	"thumbnail": "https://chompubot.work/bot.png",
```

Comment:
The [thumbnail](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#thumbnail) should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.